### PR TITLE
fix(Apple): fixed imgur links not working

### PR DESCRIPTION
**Pull request by Yan-Jobs**
Signed-off-by: Yan <yan@yanjobs.me>

### Fixed this: 
SCR-20221027-fxl

Old imgur links:
<details>

LOGO
THUMBNAIL

</details>

New ones:
<details>

test
test

</details>

## Acknowledgements
- [x] I read the [Presence Guidelines](https://github.com/PreMiD/Presences/blob/main/.github/CONTRIBUTING.md)
[NOT APPLICABLE] I linted the code by running `yarn format`
- [x] The PR title follows the repo's [commit conventions](https://github.com/PreMiD/Presences/blob/main/.github/COMMIT_CONVENTION.md)

## Screenshots
<details>
<summary> Proof showing the creation/modification is working as expected </summary>

I can't show this, since it's only a metadata change and I can't change the official website.

</details>

### CODE REVIEW

**Comment by Yan-Jobs** [comment]
Version bump isn't mandatory because it is ONLY a metadata change, indeed a new version is useless

**Comment by EncryptedDev** [comment]
> Version bump isn't mandatory because it is ONLY a metadata change, indeed a new version is useless

Who says a version bump isn’t needed? I’m not sure you know

File: websites/A/Apple/metadata.json [hunk 1]
```diff
@@ -8,6 +8,10 @@
 		{
 			"name": "Drull",
 			"id": "244953147154366464"
+		},
+		{
+			"name": "Yan.",
+			"id": "735538297815957584"
```
Comment (N0chteil) [inline websites/A/Apple/metadata.json:14]:
```suggestion
		}
```

You didn't edit anything of the codebase, so please remove this

https://github.com/PreMiD/Presences/blob/main/.github/CONTRIBUTING.md
> If you make modifications to a presence and change at **least a quarter** of the presence's **codebase**, you are allowed to add yourself as a contributor. Contact a reviewer for more information about this subject.

File: websites/A/Apple/metadata.json [hunk 2]
```diff
@@ -29,8 +33,8 @@
 		"www.icloud.com"
 	],
 	"version": "2.3.1",
-	"logo": "https://i.imgur.com/rBAusA5.png",
-	"thumbnail": "https://i.imgur.com/D5b7fQ5.jpg",
+	"logo": "https://i.imgur.com/w84AMlL.png",
+	"thumbnail": "https://i.imgur.com/83wQlSy.jpeg",
```
Comment (N0chteil) [inline websites/A/Apple/metadata.json:37]:
```suggestion
	"thumbnail": "https://i.imgur.com/MZecshr.jpg",
```
image

The thumbnail doesn't fit. We should use this one instead:
image

**Review by N0chteil — CHANGES_REQUESTED** [review]

**Review by EncryptedDev — APPROVED** [review]

**Review by Bas950 — APPROVED** [review]

**Pull request merged**